Source/CesiumRuntime/Public/CesiumGeoreference.h

```diff
@@ -84,9 +98,30 @@ class CESIUMRUNTIME_API ACesiumGeoreference : public AActor {
   UPROPERTY(BlueprintAssignable, Category = "Cesium")
   FGeoreferenceUpdated OnGeoreferenceUpdated;
 
+  /**
+   * An event that will be called whenever the georeference's ellipsoid has
+   * been modified.
+   */
+  UPROPERTY(BlueprintAssignable, Category = "Cesium")
+  FGeoreferenceEllipsoidChanged OnEllipsoidChanged;
```

Comment:
Adding a new event when the ellipsoid is changed makes sense, but this makes me think that `OnGeoreferenceUpdated` isn't really pulling its weight.

Have you considered modifying `OnGeoreferenceUpdated` to pass the old / new ellipsoid to the callers? In theory, we can just have the one event, and it can serve its existing callers, but also support what `ACesium3DTileset` needs (`RefreshTileset` when the ellipsoid changes)

Reply:
The reason there's two events is that there's two different things that need to happen. One is to update a globe anchor's position when the georeference changes in a way that would affect its computations, the other is to unload and reload a tileset when the ellipsoid has changed. We only need to do one or the other - because GlobeAnchor stores its positions in ECEF coordinates, its position isn't going to change when the ellipsoid is changed. And we definitely don't want to unload and reload the tileset whenever the georeference is updated. So either we have one event with an enum or bool value that specifies which of the two operations to perform - or we have two distinct events.

Reply:
> So either we have one event with an enum or bool value that specifies which of the two operations to perform

Right, that's what I was implying that we could do instead. Both approaches seem valid though.